Makefile

```diff
@@ -248,7 +249,6 @@ validate-schemas:
 	@$(ROOT)/scripts/validate-schemas.sh $(ROOT)/applications/crossbar/priv/couchdb/schemas
 
 CHANGED := $(shell git --no-pager diff --name-only HEAD origin/master -- applications core scripts)
-TO_FMT := $(shell git --no-pager diff --name-only HEAD origin/master -- "*.erl" "*.hrl" "*.escript")
```

Comment:
Why remove this? By default, format the project Erlang files, unless overridden explicitly in the `make fmt` invocation.

Reply:
I re-added `TO_FMT`.